src/obc/providers/obf.py

```diff
@@ -249,13 +250,35 @@ class BadgeIssue(BaseModel):
     email_body: Optional[str] = None
     email_link_text: Optional[str] = None
     email_footer: Optional[str] = None
-    badge_override: Optional[IssueBadgeOverride] = None
-    log_entry: Optional[dict] = None
+    badge_override: Optional[IssueBadgeOverride | str] = None
+    log_entry: Optional[dict | str] = None
```

Comment:
I think it's more reliable to use Pydantic's `Json` type:

```suggestion
    log_entry: Optional[dict | Json[Any]] = None
```

Reply:
Indeed good catch! And with this type, no need to deserialize it to a dict as it's done by pydantic